.circleci/script/deploy_dags/master_dag.py

```diff
@@ -0,0 +1,150 @@
+import time
+from datetime import datetime
```

Comment:
As per kaxil suggestion it will be better if we use 
`from airflow.utils.timezone import datetime` instead of `from datetime import datetime`